datagouv/base_object.py

```diff
@@ -70,6 +78,27 @@ def delete_extras(self, payload: dict) -> httpx.Response:
         self.refresh()
         return r
 
+    @simple_connection_retry
+    def get_metrics(

```

Comment:
I think it may be confusing since there are existing metrics on the object (followers, total views, discussions, etc.).
This method should be more explicit? Like get montly traffic metrics?